frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -57,7 +57,7 @@ const makeOCSRequest = (
   isEncrypted: boolean,
   isMinimal: boolean,
 ): Promise<any> => {
-  const promises = labelNodes(selectedNodes);
+  const promises: Promise<K8sResourceKind>[] = [...labelNodes(selectedNodes), labelOCSNamespace()];
```

Comment:
This is a stale file. We are waiting for 4.6 testing and backports to finish to get rid of this.
```suggestion
  const promises: Promise<K8sResourceKind>[] = [...labelNodes(selectedNodes)];
```
Additionally having a comment that its "Not to be used for 4.7+" would be good.
Its not in scope of the PR, hence wont enforce!

Reply:
Thanks for pointing it out. Since it's stale I will keep this file untouched.